src/screens/Home/components/ChooseLocation/ChooseLocation.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Box, Text } from 'grommet'
+import styled from 'styled-components'
+import withThemeContext from 'helpers/withThemeContext'
+import Map from 'images/hover-map.png'
+import ResponsiveImage from 'components/ResponsiveImage'
+import MapLabel from './components/MapLabel'
+import theme from './theme'
+
+const BasisBox = styled(Box)`
+  flex: 1 1 175px;
+`
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+const OPTIONS = [
+  { label: 'Falkland Islands', map: Map},
+  { label: 'Baja, California', map: Map},
+  { label: 'Tasmania, Australia', map: Map}
+]
+
+function ChooseLocation() {
+  const [activeMap, setActiveMap] = React.useState(null)
+
+  return (
+    <Box direction='row' gap='small' wrap>
+      <BasisBox fill='horizontal' width={{ min: '50%' }}>
+        <StyledText
+          margin='xxsmall'
+          size='1.25rem'
+        >
+          Choose a location to begin
+        </StyledText>
+        {OPTIONS.map((location, i) => {
+          return (
+            <MapLabel
+              key={`MAP_LABEL_${i}`}
+              location={location.label}
+              map={location.map}
+              onActivate={setActiveMap}
+            />
+        )})}
+      </BasisBox>
+      <BasisBox align='start'>
+        {activeMap && <ResponsiveImage border src={activeMap} />}
+      </BasisBox>
+    </Box>
```

Comment:
Same comment about trying `Grid` rather than flex box here.

Reply:
I don't think this section lends itself to `Grid` as easily as the home page since I'm trying to keep boxes wrapping at a certain width. Also, I think `Grid` requires an even number of rows and columns (albeit with spanning across rows/columns allowed) and this is strictly two columns that wrap.